CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 All notable changes to the LaunchDarkly Flutter client-side SDK will be documented in this file. This project adheres to [Semantic Versioning](https://semver.org).
 
+## [2.0.3] - 2023-08-09
+### Fixed:
+- Fixes evaluation detail ruleIndex bug affecting iOS
+
+## [2.0.2] - 2023-06-16
+### Fixed:
+- Fixes threading issue in start routine in Android native code
```

Comment:
For reviewers: This changelog did not propagate correctly last release, fixing now.